### DISCUSSION TITLE
Need help setting up my meter.

### DISCUSSION THREAD

**Discussion by brdlydupuy12** [jomjol/AI-on-the-edge-device#3419] [Q&A]
Got AI on the edge setup, but I am having hard time understanding the different modules and ROIs. I have attached a picture of my meter and the references I currently have setup. Any assistance would be greatly appreciated!!!
![water_meter](https://github.com/user-attachments/assets/5f116078-9d6b-44a0-93f7-0cb3cc664fab)

**Comment by SybexX** [comment]
1. try to get better image quality by playing around with the camera settings

2. try to get rid of the reflection/mirroring https://github.com/jomjol/AI-on-the-edge-device/discussions/1949
![391826420-5f116078-9d6b-44a0-93f7-0cb3cc664fab](https://github.com/user-attachments/assets/23a797cb-0ff5-4f6e-8cfb-26a82074808a)

3. please note the instructions from the wiki when placing the ROIs https://jomjol.github.io/AI-on-the-edge-device-docs/ROI-Configuration/
![391826420-5f116078-9d6b-44a0-93f7-0cb3cc664fab1](https://github.com/user-attachments/assets/dbe4613c-b30b-410f-80e7-817b4683ff9b)

Also note that you will occasionally have problems with the evaluation, namely if the pointer covers the numbers.
Possibly also when aligning the image(when the pointer is over the alignment marks), so you should align the camera yourself correctly and set AlignmentAlgo to off. https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-alignmentalgo
